usaspending_api/common/validator/award_filter.py

```diff
@@ -4,6 +4,7 @@
 
 from usaspending_api.awards.v2.lookups.lookups import award_type_mapping
 from usaspending_api.common.validator.helpers import TINY_SHIELD_SEPARATOR
+from usaspending_api.search.elasticsearch.filters.tas import TasCodes, TreasuryAccounts
```

Comment:
I understand why these imports into the global TinyShield rule models were done. Still, it seems to be a slippery slope on the past of cluttering this file with imports when the goal of the file is simply defining a few dictionaries. Thoughts?

Reply:
My first thought would be that the danger of magic strings dwarfs any concern from having too many import statements at the top. Obviously I can't speak to whatever happened in the past, but if something is in two places I think another import is a small price to pay to keep code DRY.

Reply:
Oops, ~past~ => `path`.

My typo misdirected the reader to look back when I'm thinking forward. To be logically consistent, all of the new Elasticsearch filter objects would be imported into this file to reuse the underscore_name.

Reply:
That sounds like a good thing.